aclk/aclk_stats.c

```diff
@@ -261,6 +261,28 @@ static void aclk_stats_newproto_rx(uint32_t *rx_msgs_sample)
 }
 #endif
 
+static void aclk_stats_mqtt_wss(struct mqtt_wss_stats *stats)
+{
+    static RRDSET *st = NULL;
+    static RRDDIM *rd_sent = NULL;
+    static RRDDIM *rd_recvd = NULL;
+
+    if (unlikely(!st)) {
+        st = rrdset_create_localhost(
+            "netdata", "aclk_openssl_written", NULL, "aclk", NULL, "Sent bytes.", "B/s",
+            "netdata", "stats", 200011, localhost->rrd_update_every, RRDSET_TYPE_STACKED);
+
+        rd_sent  = rrddim_add(st, "sent", NULL, 1, localhost->rrd_update_every, RRD_ALGORITHM_ABSOLUTE);
+        rd_recvd = rrddim_add(st, "received", NULL, 1, localhost->rrd_update_every, RRD_ALGORITHM_ABSOLUTE);
```

Comment:
This should be INCREMENTAL not ABSOLUTE so that you just keep a total of the bytes sent received and not reset manually.  The multiplier on the received should also be `-1` to achieve the `below the X axis` result